example/mapbox.html

```diff
@@ -38,6 +38,7 @@
             min-height: 40px;
             height: auto;
             max-height: 400px;
+            font-family: monospace;
```

Comment:
Great! a monospaced font is really needed, but, can we set a larger font size too?

Reply:
Yes. Is 14px ok for you?

Reply:
Thanks!